packages/components/src/components/cfd-compare-accounts-carousel/cfd-compare-accounts-carousel.tsx

```diff
@@ -34,10 +36,20 @@ const CFDCompareAccountsCarousel = (props: TCFDCompareAccountsCarousel) => {
     return (
         <div className='cfd-compare-accounts-carousel'>
             <div className='cfd-compare-accounts-carousel__viewport' ref={emblaRef}>
-                <div className='cfd-compare-accounts-carousel__container'>{props.children}</div>
+                <div className='cfd-compare-accounts-carousel__container'>{children}</div>
             </div>
-            <CFDCompareAccountsCarouselButton onClick={scrollPrev} isNext={false} enabled={prev_btn_enabled} />
-            <CFDCompareAccountsCarouselButton onClick={scrollNext} isNext={true} enabled={next_btn_enabled} />
+            <CFDCompareAccountsCarouselButton
+                onClick={scrollPrev}
+                isNext={false}
```

Comment:
Can be omitted :)
```suggestion
```

Reply:
@maryia-deriv No, it can't :) Of course I can modify props of `CFDCompareAccountsCarouselButton`, but this prop is required, but optional... But I'll update it